source/analysis/AM2DSummingAB.cpp

```diff
@@ -315,40 +317,92 @@ bool AM2DSummingAB::axisValues(int axisNumber, int startIndex, int endIndex, dou
 
 	int otherAxis = (sumAxis_ == 0) ? 1 : 0;
 
-	if (startIndex >= inputSource_->axisInfoAt(otherAxis).size || endIndex >= inputSource_->axisInfoAt(otherAxis).size)
+	if (startIndex >= inputSource_->size(otherAxis) || endIndex >= inputSource_->size(otherAxis))
 		return false;
 
 	return inputSource_->axisValues(otherAxis, startIndex, endIndex, outputValues);
 }
 
+void AM2DSummingAB::setSumAxis(int sumAxis)
+{
+	if((unsigned)sumAxis >= 2)
```

Comment:
Good catch.  That is a legacy bug as I mostly just moved these functions from the header to the cpp file.